tests/scripts/init-vault.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-vault server -dev -dev-root-token-id="${VAULT_TOKEN}" &
+vault server -dev \
+    -dev-listen-address="${VAULT_ADDR#http://}" \
```

Comment:
I figured out the issue... the comment on this line was causing the command to be interpreted incorrectly :( lol